inlanecruising_plugin/include/inlanecruising_plugin/object_avoidance.h

```diff
@@ -0,0 +1,87 @@
+#pragma once
+
+/*
+ * Copyright (C) 2019-2020 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#include <vector>
+#include <cmath>
+#include <cav_msgs/TrajectoryPlan.h>
+#include <cav_msgs/TrajectoryPlanPoint.h>
+#include <cav_msgs/Plugin.h>
+#include <trajectory_utils/trajectory_utils.h>
+#include <carma_utils/CARMAUtils.h>
+#include <geometry_msgs/PoseStamped.h>
+#include <geometry_msgs/TwistStamped.h>
+#include <cav_srvs/PlanManeuvers.h>
+#include <cav_srvs/PlanTrajectory.h>
+
+#include <carma_wm/WMListener.h>
+#include <carma_wm/WorldModel.h>
+#include <carma_wm/collision_detection.h>
+#include <trajectory_utils/quintic_coefficient_calculator.h>
+
+
+
+namespace inlanecruising_plugin
+{
+namespace object_avoidance
+{
+
+    class ObjectAvoidance
```

Comment:
Member variables here should have the `_` marking them as such. Might be worth considering which can be made const as well, or which might be good as params.

Reply:
added _ and const. These are params in Yield, but not here, since it would include making changes to the constructor and also more cleanup after removing.